en/docs/tutorials/start-developing-an-integration-with-a-managed-rest-api.md

```diff
@@ -0,0 +1,434 @@
+# Start Developing an Integration with a Managed API
+
+The top down approach of API first integration provides the capability for API developers to start creating a managed REST API first in WSO2 API Manager. Then the integration developer can use the same API in the WSO2 Integration Studio to develop the integration later on and expose it to WSO2 API Manager. Thereafter, the API consumers can discover the API from the marketplace, subscribe to it, and use it for application development.
+
+The following diagram demonstrates how the integration components and API management components of WSO2 API Manager work together to enable API-led integration when working with the Top Down approach. Furthermore, this diagram also illustrates the various API Manager components and the different user roles that are involved in implementing this process:
+
+**Stakeholders**
+
+- API Creator
+- Integration developer
+- API Publisher
+- API Consumer
+
+## What you'll build
+
+Let's work with the following user story in this tutorial.
+
+The API developer has a Swagger definition that requires the creation of an API to be managed and some integrations. The API developer creates a prototype API via the Control Plane (Publisher Portal), creates mock implementations, and tests the API. The integration developer picks up the API definition, implements the integrations and deploys it in the Micro Ingegrator server. Thereafter, the API developer adds the service endpoint using API Manager and publishes the prototyped API.
+
+Let's get started!
+
+## Before you begin
```

Comment:
Shall we move this to 11 th step ?

Reply:
fixed in 00fc111ebc487d098466c5f040eb785b9a15fc3e